src/test/unit/cli/cliExecutable.test.ts

```diff
@@ -10,30 +11,60 @@ suite('CliExecutable', () => {
   });
 
   test('Returns correct filename for different platforms', () => {
-    strictEqual(CliExecutable.getFilename('linux'), 'snyk-linux');
-    strictEqual(CliExecutable.getFilename('darwin'), 'snyk-macos');
-    strictEqual(CliExecutable.getFilename('win32'), 'snyk-win.exe');
+    strictEqual(CliExecutable.getFileName('linux'), 'snyk-linux');
+    strictEqual(CliExecutable.getFileName('linux_alpine'), 'snyk-alpine');
+    strictEqual(CliExecutable.getFileName('macos'), 'snyk-macos');
+    strictEqual(CliExecutable.getFileName('macos_arm64'), 'snyk-macos-arm64');
+    strictEqual(CliExecutable.getFileName('windows'), 'snyk-win.exe');
   });
 
-  test('Returns correct extension paths', () => {
+  test('Returns correct extension paths', async () => {
     const unixExtensionDir = '/Users/user/.vscode/extensions/snyk-security.snyk-vulnerability-scanner-1.1.0';
+    const winExtensionDir = `C:\\Users\\user\\.vscode\\extensions`;
+
+    const osStub = sinon.stub(Platform, 'getCurrent').returns('darwin');
+    const archStub = sinon.stub(Platform, 'getArch').returns('x64');
+    const fsStub = sinon.stub(fs, 'access').returns(Promise.reject());
 
-    const stub = sinon.stub(Platform, 'getCurrent').returns('darwin');
     let expectedCliPath = path.join(unixExtensionDir, 'snyk-macos');
-    strictEqual(CliExecutable.getPath(unixExtensionDir), expectedCliPath);
+    strictEqual(await CliExecutable.getPath(unixExtensionDir), expectedCliPath);
 
-    stub.returns('linux');
+    osStub.returns('linux');
     expectedCliPath = path.join(unixExtensionDir, 'snyk-linux');
-    strictEqual(CliExecutable.getPath(unixExtensionDir), expectedCliPath);
+    strictEqual(await CliExecutable.getPath(unixExtensionDir), expectedCliPath);
 
-    const winExtensionDir = `C:\\Users\\user\\.vscode\\extensions`;
-    stub.returns('win32');
+    fsStub.returns(Promise.resolve());
+    expectedCliPath = path.join(unixExtensionDir, 'snyk-alpine');
+    strictEqual(await CliExecutable.getPath(unixExtensionDir), expectedCliPath);
+    fsStub.returns(Promise.reject());
+
+    osStub.returns('win32');
+    expectedCliPath = path.join(winExtensionDir, 'snyk-win.exe');
```

Comment:
nitpick(non-blocking): I think arm64 for windows is missing in the tests? I know it works...